### PR TITLE
Improve readability of debug menu by using '|'

### PR DESCRIPTION
**Pull request by ClobberXD**
screenshot_20190426_221014

### CODE REVIEW

**Comment by DS-Minetest** [comment]
I have doubts that this is really more readable.

**Comment by paramat** [comment]
debug

Master for comparison. I'm unsure it's an improvement.

**Comment by paramat** [comment]
The current use of commas makes everything somewhat run into the next. I find the PR an improvement.

**Comment by ClobberXD** [comment]
@DS-Minetest The `|` is always a great candidate for separators, as it spans the whole height of the line, and is also (generally) surrounded by spaces on both sides.

I myself should've also added a screenshot of the debug menu in master for comparison. Thanks, paramat.

**Comment by ClobberXD** [comment]
I noticed that yaw and cardinal direction don't have a separator in between. Are they considered to be related, or should I add a separator there?

File: src/client/gameui.cpp [hunk 1]
```diff
@@ -126,10 +126,10 @@ void GameUI::update(const RunStats &stats, Client *client, MapDrawControl *draw_
 			<< "pos: (" << (player_position.X / BS)
 			<< ", " << (player_position.Y / BS)
 			<< ", " << (player_position.Z / BS)
-			<< "), yaw: " << (wrapDegrees_0_360(cam.camera_yaw)) << "° "
+			<< ") | yaw: " << (wrapDegrees_0_360(cam.camera_yaw)) << "°"
```
Comment (Thomas--S) [inline src/client/gameui.cpp]: If you are not going to add a separator here, I suggest to keep at least the space as it is in master.

```suggestion
			<< ") | yaw: " << (wrapDegrees_0_360(cam.camera_yaw)) << "° "
```

Comment (ClobberXD) [reply src/client/gameui.cpp]: Will do, if reviewers don't see the need to separate yaw and cardinal direction.

Comment (ClobberXD) [reply src/client/gameui.cpp]: Restored the space. Thanks.

**Comment by paramat** [comment]
Yaw, NSEW and +_XZ are all closely related so are in one section, no need to change that.

**Pull request merged**